src/Spinner.class.ts

```diff
@@ -71,15 +75,11 @@ export class Spinner extends EventEmitter {
       return;
     }
 
-    const { name = kDefaultSpinnerName, color = null } = options;
+    const { name = kDefaultSpinnerName, color = "white" } = options;
 
     this.#spinner = name in cliSpinners ? cliSpinners[name] : cliSpinners[kDefaultSpinnerName];
-    if (color === null) {
-      this.#color = (str: string) => str;
-    }
-    else {
-      this.#color = color in kleur ? kleur[color] : kleur.white;
-    }
+
+    this.#color = kAvailableColors.has(color.toString()) ? color : "white";
```

Comment:
If an array is given (and has multiple items), it will fallback to `white`.
I think we can do `this.#color = color` since `color` is typed and has a default value.
If we wanna check the value, something like that maybe:
```ts
const colors = Array.isArray(color) ? color : [color];
if (colors.every((color) => kAvailableColors.has(color)) === false) {
  throw new Error("Invalid color given")
}  
```
or
```ts
const colors = Array.isArray(color) ? color : [color];
const invalidColors = colors.filter((color) => kAvailableColors.has(color) === false);
if (invalidColors.length) {
  throw new Error(`Invalid color(s): ${invalidColors}`;
}
```